src/main/java/org/assertj/core/api/InstanceOfAssertFactories.java

```diff
@@ -263,6 +263,12 @@
   InstanceOfAssertFactory<char[], AbstractCharArrayAssert<?>> CHAR_ARRAY = new InstanceOfAssertFactory<>(char[].class,
                                                                                                          Assertions::assertThat);
 
+  /**
+   * {@link InstanceOfAssertFactory} for a {@code char} two-dimensional array.
+   */
+  InstanceOfAssertFactory<char[][], Char2DArrayAssert> CHAR_2D_ARRAY = new InstanceOfAssertFactory<>(char[][].class,
+                                                                                                         Assertions::assertThat);
```

Comment:
formatting is off, Assertions::assertThat should be aligned with the first param, make sure your IDE follows our formatting convention

Reply:
Sorry, I had different formatter set for this project. Fixed thant and reformatted the code with formatter used in AssertJ.